src/helpers/MetricDatasourceHelper.ts

```diff
@@ -251,19 +236,19 @@ export class MetricDatasourceHelper {
    * ```
    */
   public static fetchLabels(params: FetchLabelsOptions): Promise<string[]> {
-    const { ds, timeRange, matcher } = params;
+    const { timeRange, matcher } = params;
+    const ds = params.ds as PrometheusRuntimeDatasource;
```

Comment:
```suggestion
    const ds = params.ds;
```

The type assertion is not necessary if we modify the `FetchLabelsOptions` below. And we can do the same for `FetchLabelValuesOptions`.